### DISCUSSION TITLE
사용자 유치 계획 회의

### DISCUSSION THREAD

**Discussion by Kim0914** [woowacourse-teams/2023-pium#328] [뜨거운 토론]
## 사용자 유치 목표

* 피움 : 100명 (공략 대상이 일반적인 사용자이거나, 명확한 경우)

## 어떻게 How?

* **민간협력컨설팅과 같은 지자체와 협조** -> 이벤트 성으로 그치지 않고, 식물 클래스 참여자와 같은 꾸준한 사용자를 유치
* 지자체에서 시행하는 원예 프로그램을 진행하며 자연스럽게 어플 홍보
* 인스타그램에 식집사, 식물스타그램 등 식물 태그들을 파악한 후 DM으로 홍보
* **대학교 식물 동아리 -> 신뢰성 있는 사람들**에게 홍보
* 조경과, 원예과와 같은 곳에 사용을 의뢰
* 식물 커뮤니티 홍보
* 식물을 파는 가게에 홍보
* 피움 인스타그램을 이용한 홍보
* 네이버 밴드, 블로그, 인스타그램 릴스, 유튜브 쇼츠를 통한 홍보

## 가장 먼저 실행해 볼 계획
* 인스타그램에 `식집사`, `식물스타그램` 등 식물 태그들을 파악한 후 DM으로 서비스 소개 후 피드백 요청
* 대학교 식물 동아리와 같은 신뢰성 있는 단체에게 서비스 소개 후 피드백 요청
* 조경과, 원예과와 같은 곳에 서비스 소개 후 피드백 요청

## 차후 실행해 볼 계획
* 식물 커뮤니티 홍보
* 식물을 파는 가게에 홍보
* 피움 인스타그램을 이용한 홍보
* 네이버 밴드, 블로그, 인스타그램 릴스, 유튜브 쇼츠를 통한 홍보
* 민간협력컨설팅과 같은 지자체와 협조 -> 이벤트 성으로 그치지 않고, 식물 클래스 참여자와 같은 꾸준한 사용자를 유치 **(실현 가능성에 대한 고민 필요)**
* 지자체에서 시행하는 원예 프로그램을 진행하며 자연스럽게 어플 홍보 **(실현 가능성에 대한 고민 필요)**

## 유지 계획
* 관리자 페이지를 통해 사전 식물 등록 요청 기능 추가 -> 사용자가 기르는 식물이 없는 경우 즉각적으로 대응
* 피움 채널톡을 운영해 소통 창구 -> 채널톡이 제한되면 구글 폼을 우선적으로 사용
* 각 식물 마다 사용자가 **반려 식물 일기와 같은 기록을 남길 수 있도록** 기능 구현

**Comment by yeonkkk** [comment]
- 인스타그램 카드 뉴스 추가하기 : 피움 사용자 가이드

**Comment by Choi-JJunho** [comment]
유치계획 제출 및 수행 완료로 close 하겠습니다.

- https://github.com/woowacourse/retrospective/discussions/40#discussioncomment-6910311
- [피움 인스타그램 카드뉴스](https://www.instagram.com/p/Cw1hl-RSdap/)
